Update download_and_installation.rst (#5697)
ArduPilot/ardupilot_wiki: mavproxy/source/docs/getting_started/download_and_installation.rst

.. note::
    If MAVProxy is intended to run "headless" (non-GUI), the ``python3-wxgtk4.0`` or ``python3-wxpython4`` packages can be skipped.

>     If for some reason 'module load console' fails, then it means u don't have all the required GUI prerequisites [typically wx], and 'pip3 install --upgrade WxPython' might help you , this can be slow as it has to compile a bunch of stuff for wx/python integration.

Reviewer: isnt this specified immediately above in the first install command?

>     If for some reason 'module load console' fails, then it means u don't have all the required GUI prerequisites [typically wx], and 'pip3 install --upgrade WxPython' might help you , this can be slow as it has to compile a bunch of stuff for wx/python integration.

Reviewer: ```suggestion
    If for some reason 'module load console' fails, then it it may mean you don't have all the required GUI prerequisites [typically wx], and executing 'pip3 install --upgrade WxPython' might help you , this can be slow as it has to compile a lot for wx/python integration.

```

Reviewer 2: this won't work in most case as you need a bunch of dependencies to compile wx and if you have it install with apt-get, pip won't erase it normally...

Reviewer 2: yep, pip + compilation ins't great. On debian/ubuntu, apt install should work in most case. Then we should hope that pip got a wheels for wx as compilation is a pain and shouldn't be proposed to a standard user

Reviewer: @khancyr  @davidbuzz shall we close this?

Reviewer 2: yep, better not advice this

Reviewer: @davidbuzz I am going to close this unless we get a convincing argument
